Build fails because of uninitialized variable in rootston

wlroots version: [`c168665043e7c0585652ff414793e44351fc72de`](https://github.com/swaywm/wlroots/commit/c168665043e7c0585652ff414793e44351fc72de)

wlroots configuration:

```
% meson build                      \
        --backend=ninja            \
        --buildtype=release        \
        --prefix=/usr              \
        --sysconfdir=/etc          \
        --localstatedir=/var       \
        --mandir=/usr/share/man    \
        --infodir=/usr/share/info  \
        -Denable-libcap=true       \
        -Denable-systemd=true      \
        -Denable-elogind=false     \
        -Denable-xcb_errors=false  \
        -Denable-xwayland=true     \
        -Denable-x11_backend=false
The Meson build system
Version: 0.45.1
Source dir: /home/me/Projects/wlroots
Build dir: /home/me/Projects/wlroots/build
Build type: native build
Project name: wlroots
Native C compiler: cc (gcc 7.3.1 "cc (GCC) 7.3.1 20180312")
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: /usr/bin/pkg-config (0.29.2)
Native dependency wayland-server found: YES 1.14.0
Native dependency wayland-client found: YES 1.14.0
Native dependency wayland-egl found: YES 17.3.7
Native dependency wayland-protocols found: YES 1.13
Native dependency egl found: YES 17.3.7
Native dependency glesv2 found: YES 17.3.7
Native dependency libdrm found: YES 2.4.91
Native dependency gbm found: YES 17.3.7
Native dependency libinput found: YES 1.10.3
Native dependency xkbcommon found: YES 0.8.0
Native dependency libudev found: YES 238
Native dependency pixman-1 found: YES 0.34.0
Native dependency libcap found: YES 2.25
Native dependency libsystemd found: YES 238
Dependency libelogind found: NO
Library m found: YES
Native dependency xcb found: YES 1.13
Native dependency xcb-composite found: YES 1.13
Native dependency xcb-xfixes found: YES 1.13
Native dependency xcb-image found: YES 0.4.0
Native dependency xcb-render found: YES 1.13
Native dependency x11-xcb found: YES 1.6.5
Native dependency xcb-icccm found: YES 0.4.1
Native dependency xcb-xkb found: YES 1.13
Dependency xcb-errors found: NO
Configuring config.h using configuration
Configuring version.h using configuration
Program wayland-scanner found: YES (/usr/bin/wayland-scanner)
Program ../glgen.sh found: YES (/home/me/Projects/wlroots/render/../glgen.sh)
Message: 
----------------
wlroots 0.0.1

      libcap: true
     systemd: true
     elogind: false
    xwayland: true
 x11_backend: false
   xcb-icccm: true
  xcb-errors: false
----------------

Dependency threads found: YES
Native dependency wayland-cursor found: YES 1.14.0
Program git found: YES (/usr/bin/git)
Program etags found: YES (/usr/bin/etags)
Program ctags found: NO
Build targets in project: 24
Found ninja-1.8.2 at /usr/bin/ninja
```

wlroots build:

```
% ninja -C build
ninja: Entering directory `build'
[194/316] Compiling C object 'rootston/rootston@exe/desktop.c.o'.
FAILED: rootston/rootston@exe/desktop.c.o 
cc  -Irootston/rootston@exe -Irootston -I../rootston -Iinclude -I../include -I/usr/include/libdrm -I/usr/include/libevdev-1.0/ -I/usr/include/libwacom-1.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/pixman-1 -I/home/me/Projects/wlroots/build -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -O3 -Wno-unused-parameter '-DWLR_SRC_DIR="/home/me/Projects/wlroots"' -DWL_HIDE_DEPRECATED -MD -MQ 'rootston/rootston@exe/desktop.c.o' -MF 'rootston/rootston@exe/desktop.c.o.d' -o 'rootston/rootston@exe/desktop.c.o' -c ../rootston/desktop.c
../rootston/desktop.c: In function ‘desktop_surface_at’:
../rootston/desktop.c:600:5: error: ‘_surface’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
  if (_surface != NULL) {
     ^
../rootston/desktop.c:581:22: note: ‘_surface’ was declared here
  struct wlr_surface *_surface;
                      ^~~~~~~~
../rootston/desktop.c:600:5: error: ‘_surface’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
  if (_surface != NULL) {
     ^
../rootston/desktop.c:581:22: note: ‘_surface’ was declared here
  struct wlr_surface *_surface;
                      ^~~~~~~~
cc1: all warnings being treated as errors
[199/316] Compiling C object 'rootston/rootston@exe/output.c.o'.
ninja: build stopped: subcommand failed.
```

--- a/rootston/desktop.c
+++ b/rootston/desktop.c
@@ -578,7 +578,7 @@ static bool view_at(struct roots_view *view, double lx, double ly,
 	}
 
 	double _sx, _sy;
-	struct wlr_surface *_surface;
+	struct wlr_surface *_surface = NULL;
 	switch (view->type) {
 	case ROOTS_XDG_SHELL_V6_VIEW:
 		_surface = wlr_xdg_surface_v6_surface_at(view->xdg_surface_v6,

